packages/modules/cra-template-desktop-viewer/template.json

```diff
@@ -34,10 +34,10 @@
       "@bentley/ui-framework": "^2.14.0",
       "@bentley/ui-ninezone": "^2.14.0",
       "@bentley/webgl-compatibility": "^2.14.0",
-      "@itwin/desktop-viewer-react": "^1.0.16",
+      "@itwin/desktop-viewer-react": "^1.2.0",
```

Comment:
i think moving forward (after 3.0) we can just use the `latest` dist tag here

Reply:
agree, as long as we're still using the next tag for prerelease